common/src/main/java/org/broadleafcommerce/common/web/BroadleafRequestProcessor.java

```diff
@@ -261,6 +264,7 @@ protected void clearBroadleafSessionAttrs(WebRequest request) {
             request.removeAttribute(BroadleafCurrencyResolverImpl.CURRENCY_VAR, WebRequest.SCOPE_SESSION);
             request.removeAttribute(BroadleafTimeZoneResolverImpl.TIMEZONE_VAR, WebRequest.SCOPE_SESSION);
             request.removeAttribute(BroadleafSandBoxResolver.SANDBOX_ID_VAR, WebRequest.SCOPE_SESSION);
+            request.removeAttribute("blIncludeMyChanges", WebRequest.SCOPE_SESSION);
```

Comment:
let's extract blIncludeMyChanges to "constant" and re-use, similar to BroadleafSandBoxResolver.SANDBOX_ID_VAR